code/game/machinery/doors/airlock_types.dm

```diff
@@ -298,6 +298,16 @@
 	opacity = 0
 	glass = TRUE
 
+/obj/machinery/door/airlock/bronze
+	name = "bronze airlock"
+	icon = 'icons/obj/doors/airlocks/clockwork/pinion_airlock.dmi'
+	overlays_file = 'icons/obj/doors/airlocks/clockwork/overlays.dmi'
+	assemblytype = /obj/structure/door_assembly/door_assembly_bronze
+
```

Comment:
You will need to add the actual anim_part from clockcult airlocks else this is not going to look right.

anim_parts = "left=-13,0;right=13,0"
Those should be the right values